src/Language/PureScript/Docs/Types.hs

```diff
@@ -48,7 +63,6 @@ data Package a = Package
   , pkgTagTime              :: Maybe UTCTime
   , pkgModules              :: [Module]
   , pkgModuleMap            :: Map P.ModuleName PackageName
-  , pkgResolvedDependencies :: [(PackageName, Version)]
```

Comment:
I agree we should probably try to keep compatibility with `bower.json` though. In fact, after working through this, I'm thinking we should continue using your `bower-json` library.